app-backend/rest-api/schema.js

```diff
@@ -53,6 +53,8 @@ type Query {
     searchAllTweetsByKeyword(keyword: String!): TweetConnection
 }
 
+directive @aws_subscribe(mutations: [String]) on FIELD_DEFINITION
```

Comment:
this directive definition is needed for the schema to be valid

Reply:
only on mutations?

Reply:
the directive `aws_subsribe` is actually on the field `addTweet` of the `Subscription` type and `mutations` is the argument of the directive
reference: https://www.apollographql.com/docs/graphql-tools/schema-directives.html

Reply:
I don't remember if we were able to make subscriptions work with current integrations (it def works in appsync but not sure if it works in lambda + api gateway). can you please test that?

Reply:
I'm pretty sure subscriptions were never working with rest-api backend. They look for a websocket endpoint which is not available here
```
{
  "error": "Could not connect to websocket endpoint ws://localhost:4000/graphql. Please check if the endpoint url is correct."
}
```